.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "TAC"]
+	path = TAC
```

Comment:
Just a question: wouldn't it be better to have the TAC submodule under `src`?

Reply:
Yes, indeed